### PR TITLE
Site basics for MEC 2024

### PR DESCRIPTION
**Pull request by mfeustle**
Comments welcomed -- I hope this is helpful!

### CODE REVIEW

File: _conferences/2024/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: conference
+sidemenu: conference
+title: "Music Encoding Conference 2024"
+subtitle: ""
+venue: "University of North Texas, Denton, Texas"
+image: conference/2024/MEC22-header-sailboats.jpg
+date-start: 2024-05-20
+date-end: 2022-05-23
```
Comment (annakijas1) [inline _conferences/2024/index.html:9]: correct to 2024-05-23

File: _conferences/2024/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+id: about
+---
+<img src="/images/conference/2024/UNTMECTempLogo.jpg" class="img-responsive" alt="Music Encoding Conference, University of North Texas, 20-23 May 2024" style="margin-bottom:1em;" />
+<div class="mec2024-page">
+      
+    <p>The Music Encoding Conference is the annual meeting of the Music Encoding Initiative (MEI) community and all who are interested in the digital representation of music.</p>
+
+    <p>Music encoding is a critical component for fields and areas of study including computational or digital musicology, digital editions, symbolic music information retrieval, and digital libraries. This event brings together enthusiasts from various music research communities, including technologists, librarians, music scholars, and students and provides an opportunity for learning and engaging with and from each other.</p>
+
+    <p>Music Encoding Conference 2022 is hosted by the <a href="http://www.unt.edu" target="_blank">University of North Texas</a> and will take place Monday 20 - Thursday 23nd May, 2024.</p>
```
Comment (annakijas1) [inline _conferences/2024/index.html:22]: correct year to 2024

File: _conferences/2024/index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    
+    
+    <p>The <a href="/conference/2022/call/">Call for Contributions</a> is now available. Further details will be announced here.</p>
```
Comment (annakijas1) [inline _conferences/2024/index.html]: correct page URL for cpf to /conference/2024/call/

File: _conferences/2024/index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  
+</div>
+<!-- cf. https://developers.google.com/search/docs/data-types/event?hl=de#integrate -->
+<script type="application/ld+json">
+    {
+        "@context": "https://schema.org",
+        "@type": "Event",
+        "name": "Music Encoding Conference 2022",
+        "description": "The Music Encoding Conference is the annual meeting of the Music Encoding Initiative (MEI) community and all who are interested in the digital representation of music.",
+        "startDate": "2022-05-19",
```
Comment (annakijas1) [inline _conferences/2024/index.html:40]: correct to 2023-05-20

File: _conferences/2024/index.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        "endDate": "2022-05-22",
```
Comment (annakijas1) [inline _conferences/2024/index.html:41]: correct to 2024-05-23

File: _conferences/2024/index.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    <p>Questions? Contact Organizing Committee Chair, Maristella Feustle, or Programme Committee Chair, Margrethe Bue, at conference2024 at music-encoding.org.</p>
+
+  
+</div>
+<!-- cf. https://developers.google.com/search/docs/data-types/event?hl=de#integrate -->
+<script type="application/ld+json">
+    {
+        "@context": "https://schema.org",
+        "@type": "Event",
+        "name": "Music Encoding Conference 2022",
```
Comment (annakijas1) [inline _conferences/2024/index.html:38]: correct to 2024

File: _conferences/2024/index.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+        "startDate": "2022-05-19",
+        "endDate": "2022-05-22",
+        "eventAttendanceMode": "https://schema.org/MixedEventAttendanceMode",
+        "eventStatus": [
+            "https://schema.org/EventScheduled",
+            "https://schema.org/EventMovedOnline"
+        ],
+        "location": [{
+            "@type": "VirtualLocation",
+          	"url": "https://music-encoding.org/conference/2022/"
```
Comment (annakijas1) [inline _conferences/2024/index.html:49]: correct URL to 2024

File: _conferences/2024/index.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+        },
+            {
+                "@type": "Place",
```
Comment (annakijas1) [inline _conferences/2024/index.html:52]: correct place info

File: _conferences/2024/index.html [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+                "name": "Dalhouse University",
```
Comment (annakijas1) [inline _conferences/2024/index.html:53]: Correct university info

**Comment by mfeustle** [comment]
Ah! Sorry about those. I'll fix them and try again!

**Comment by annakijas1** [comment]
@bwbohl @musicEnfanthen please take a look at the pages for MEC 2024 and merge pull request if everything looks good. I think the menu file still needs to be updated.

**Comment by bwbohl** [comment]
thx for the PR, will have a look later today ;-)

File: _conferences/2024/call.html [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+        <li>machine learning approaches.</li>
+    </ul>
+
+
+    <h4 id="submissions">Submissions</h4>
+    <p> MEC 2024 will be accepting submissions in the following formats for the main conference programme (all submissions should include a short abstract, keywords and bio):</p>
+    <p>
+<ul>
+  <li><strong>Long papers:</strong> abstract 2-4 pages (1000-2000 words).</li>
+  <p>Long papers are expected to present overviews or detail specific aspects of ongoing or completed projects, detailed case-studies or elaborated perspectives on best practices in the field, or provide other reports on topics relevant to the conference.</p>
```
Comment (musicEnfanthen) [inline _conferences/2024/call.html:74]: Technically, `p` is not allowed as child of `ul` in HTML. There are different solutions possible.

a) Remove the paragraph tags and include all the statements about long papers inside the `li` element.
b) Move the closing `li` tag to the end of the paragraphs, like this:
```
    <li>
      <strong>Long papers:</strong> abstract 2-4 pages (1000-2000 words.)
      <p>Long papers are expected to present overviews or detail specific aspects of ongoing or completed projects, detailed case-studies or elaborated perspectives on best practices in the field, or provide other reports on topics relevant to the conference.</p>
      <p>Speakers will be given 30 minutes each: 20 minutes for presentation, 10 minutes for discussion.</p>
    </li>
```

Repeat for all following list items

File: _conferences/2024/call.html [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+  <p>Panels are suited to coordinated approaches or discussions relating to a single theme. Submissions should describe the topic and nature of the panel, along with the main theses and objectives of the proposed contributions. Panel sessions will be given 90 minutes, which can be used flexibly to include, for example, 3 individual papers followed by questions, or a roundtable discussion. 
+</p>
+   <li><strong>Posters:</strong> 1-2 pages (500-1000 words).</li>
+  <p>Posters are expected to report on early-stage work, introduce new work, projects, or software, or present experimental ideas for community feedback. A “poster slam” session will be dedicated to poster presentations of 1 minute each. Subsequently, poster presenters will have the chance to tell interested parties more about their project during the poster exhibition, where the audience can browse freely. 
+</p>
+   <li><strong>Half- or full-day workshops:</strong> 2-3 pages (1000-1500 words).</li>
+  <p>Proposals must include conveners, a description of the workshop’s objective and proposed duration, as well as its logistical and technical requirements.
+Proposals must include:
```
Comment (musicEnfanthen) [inline _conferences/2024/call.html:87]:
```suggestion
Proposals must include:</p>
```

File: _conferences/2024/call.html [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+<ul><li>A brief outline of the proposed topic and its appeal to either the text or music encoding communities, or both</li>
+<li>The duration of the proposed workshop or seminar (half day, full day)</li>
+<li>Any special requirements (e.g., participant-supplied laptops, projector, flipchart)</li>
+<li>A list of proposed workshop leader(s) with a brief biography of each one</li></ul>
+</p>
```
Comment (musicEnfanthen) [inline _conferences/2024/call.html:92]:
```suggestion
```

Comment (musicEnfanthen) [reply _conferences/2024/call.html:92]: Closing tag moved above the `ul`.

File: _conferences/2024/UNTMECTempLogo.jpg
[the remark this answers is not in the document]
Comment (musicEnfanthen) [reply _conferences/2024/UNTMECTempLogo.jpg:1]: As a side note (and detail that can be fixed later): Please note that the logo image is a bit smaller than in previous years. This one has about 868x240, while the former had about 1240x490 (or more)

File: _conferences/2024/call.html [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
+
+ 
+
+  <h4 id="dates">Important Dates</h4>
+
+ <p> <strong>15 January:</strong> Deadline for all submissions</p>
+
+ <p> <strong>1 April:</strong> Notification of acceptance</p>
+
+ <p> <strong>(10 June: Submissions due from presenters interested in publishing their papers. Additional details will be shared closer to the conference date.)</p>
```
Comment (bwbohl) [inline _conferences/2024/call.html:106]: here's a closing strong tag missing

File: _conferences/2024/UNTMECTempLogo.jpg
[the remark this answers is not in the document]
Comment (annakijas1) [reply _conferences/2024/UNTMECTempLogo.jpg:1]: @mfeustle if you have a larger image, please replace this file or you can do it later.

File: _conferences/2024/call.html (hunk 13 above)
Comment (annakijas1) [reply _conferences/2024/call.html]: @mfeustle

File: _conferences/2024/UNTMECTempLogo.jpg
[the remark this answers is not in the document]
Comment (mfeustle) [reply _conferences/2024/UNTMECTempLogo.jpg:1]: This is a placeholder logo while we wait for something more permanent from our library's external relations department, hopefully by the end of the month.

File: _conferences/2024/call.html [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+        <li>machine learning approaches.</li>
+    </ul>
+
+
+    <h4 id="submissions">Submissions</h4>
+    <p> MEC 2024 will be accepting submissions in the following formats for the main conference programme (all submissions should include a short abstract, keywords and bio):</p>
+    <p>
+<ul>
+  <li><strong>Long papers:</strong> abstract 2-4 pages (1000-2000 words).</li>
+  <p>Long papers are expected to present overviews or detail specific aspects of ongoing or completed projects, detailed case-studies or elaborated perspectives on best practices in the field, or provide other reports on topics relevant to the conference.</p>
```
Comment (annakijas1) [inline _conferences/2024/call.html:74]: @mfeustle it doesn't look like the tags are nested correctly. Please see the note about the `li` element above.

File: _conferences/2024/UNTMECTempLogo.jpg
[the remark this answers is not in the document]
Comment (musicEnfanthen) [reply _conferences/2024/UNTMECTempLogo.jpg:1]: Thanks, perfect.

**Comment by musicEnfanthen** [comment]
@mfeustle To make things easier, I've created a PR on your fork (https://github.com/mfeustle/music-encoding.github.io/pull/1) that should have included all missing corrections.

File: _config.yml [hunk 15]
```diff
@@ -50,8 +50,8 @@ menu:
       url: '/conference'
       submenu:
           - id: upcoming
-            label: 'MEC/TEI 2023'
-            url: '/2023'
+            label: 'MEC/TEI 2024'
```
Comment (musicEnfanthen) [inline _config.yml:53]:
```suggestion
            label: 'MEC 2024'
```

Comment (musicEnfanthen) [reply _config.yml:53]: @annakijas1 Just a small correction.

**Review by musicEnfanthen — APPROVED** [review]

**Review by bwbohl — APPROVED** [review]

**Pull request merged**